ui/app/src/components/EditSiteDialog/EditSiteDialogContainer.tsx

```diff
@@ -56,7 +64,7 @@ export function EditSiteDialogContainer(props: EditSiteDialogContainerProps) {
           isSubmitting: true
         })
       );
-      update({ id, name, description }).subscribe(
+      update({ id, name, description, imageUrl: site.imageUrl }).subscribe(
```

Comment:
Must think things through here... The correct thing was to fix the type on the service, not patch here when the image is never updated on that service. The compiler is there to help, don't just try to shut it up.
```suggestion
      update({ id, name, description }).subscribe(
```